src/lib/serverless/gql.ts

```diff
@@ -0,0 +1,5 @@
+import { Gql } from '../gql/root';
```

Comment:
suggestion: move all serverless-specific code out of `src`, either to a dir under `api` that won't create new endpoints (how?) or a new dir `api-lib` alongside `api`

Reply:
done